steps/add_csv_data_to_db.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+---
+
+# Add data to your database
+
+Populating a relational database from CSV files can be tricky due to the relationships between tables. The relationships are enforced by foreign keys constraints on each table. The foreign key constraints enforce the correct relationships between tables, that is that the foreign key columns contains values that exists in the corresponding column in the referenced table. As a result the order in which rows are added to the tables in a relational database is very important.
+
+Say for example you have a football team database with tables for `Positions`, `Players` and `Teams`. Each player have a position and and each team have multiple players, the database use a `TeamPlayers` table, to link players to teams. The database have foreign key constraints to ensure that `Players` can only use positions that exists in the `Positions` table and that the `TeamPlayers` table can only cont
+
+To populate this database from a CSV file you will need to:
+
+* populate the `Positions` table;
+* populate the `Players` table using the foreign key from the Position player;
+* populate the `Teams` table;
+* populate the `TeamPlayers` table using foreign keys from the `Teams` and the `Players` table.
+
+## Database for Nelisa
+
+Once you have a proper database schema in place with foreign keys between the tables, you you will need to add the data in this order:
```

Comment:
@SteveBarnett  - I always cover this: http://database.projectcodex.co/steps/data_modelling.html
As part of a brief Database workshop I will extend the content to mention schema - or we can replace schema with Data Model.